openshift_metrics/metrics_processor.py

```diff
@@ -65,8 +71,22 @@ def _extract_gpu_info(metric_name: str, metric: Dict) -> GPUInfo:
             gpu_resource = metric["metric"].get("resource")
             node_model = metric["metric"].get("label_nvidia_com_gpu_machine")
 
+            # If GPU labels aren't found then try to determine the gpu type
+            # from the gpu node map file
```

Comment:
@knikolla once told me I should only make comments that explain why, not what. I think this comment is explaining what?

Reply:
sure, I think I have added the "why" now; it's mostly the same thing but rephrased.